photon-api/src/main/scala/com/linkedin/photon/ml/estimators/GameEstimator.scala

```diff
@@ -225,6 +231,8 @@ class GameEstimator(val sc: SparkContext, implicit val logger: Logger) extends P
     val normalizationContextsOpt = get(coordinateNormalizationContexts)
     val ignoreThreshold = getOrDefault(ignoreThresholdForNewModels)
     val numUniqueCoordinates = updateSequence.toSet.size
+    val indices = get(interceptIndices)
+
```

Comment:
Remove extra line.

Reply:
See above change - it would make both of these lines unnecessary.

In addition, we would need to add the following default value for `coordinateInterceptIndices` in `setDefaultParams`:
```
setDefault(coordianteInterceptIndices, Map().withDefaultValue(None))
```